sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebSecurityConfigurerAdapter.java

```diff
@@ -51,7 +50,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .and()
             .logout()
                 .logoutSuccessHandler(oidcLogoutSuccessHandler())
-                .and();
+                .and()
+            .addFilterBefore(new AADHandleConditionalAccessFilter(), ExceptionTranslationFilter.class);
```

Comment:
By doing this, do you mean `ExceptionTranslationFilter` will always exist in context?

Reply:
Yes, this filter handles most authorization failures and always loaded into the filterchain.